src/usecase_regression.Rmd

```diff
@@ -0,0 +1,107 @@
+# Regression
+For the regression use case we use the well-known [Boston Housing](https://www.rdocumentation.org/packages/mlbench/versions/0.4-1/topics/BostonHousing) dataset. The road map is as follows:
+
+* define the learning task  ([here](http://mlr-org.github.io/mlr-tutorial/devel/html/task/index.html)),
+* tune the model ([here](http://mlr-org.github.io/mlr-tutorial/devel/html/tune/index.html)),
+* conduct a benchmark experiment ([here](http://mlr-org.github.io/mlr-tutorial/devel/html/benchmark_experiments/index.html)) and
+* evaluate the performance of the model ([here](http://mlr-org.github.io/mlr-tutorial/devel/html/performance/index.html)).
+
+First, we have a look at a part of the data.
+```{r}
+data(BostonHousing, package = "mlbench")
+head(BostonHousing)
+```
+We have 14 variables. As before, ``medv`` is chosen to be the response variable. 
+
+### Define a task
+Let's define the regression [task](task.md).
+
+```{r}
+## Make a task
+regr.task = makeRegrTask(data = BostonHousing, target = "medv")
+regr.task$task.desc$n.feat
+```
+With the command ``regr.task$task.desc$n.feat`` the type of all variables in the data can be accessed. 
+
+### Tuning
+By calling ``listLearners("regr")`` we can see which learners are available for the regression task. We picked the classical linear regression model (``regr.lm``), the [Lasso](https://www.rdocumentation.org/packages/penalized/versions/0.9-50) regression model (``regr.glmnet``), [SVM](https://www.rdocumentation.org/packages/kernlab/versions/0.9-25) with a radial basis kernel (``regr.ksvm``) and random forest from the [ranger](https://www.rdocumentation.org/packages/ranger/versions/0.6.0) package (``regr.ranger``), which we would like to compare. Before setting up a [benchmark experiment](benchmark_experiments.md) some of the models' hyperparameters can be tuned. The [%mlr] package provides powerful tuning algorithms. See [Tuning](tuning.md) and [Advanced Tuning](advanced_tune.md) for more details.
```

Comment:
Could you please update this paragraph? It seems that in your code below, you are using slightly different learners than stated above.